views/posts/index.php

```diff
@@ -44,12 +44,16 @@
         <div>
           <div class="actions">
             <div class="info">
-              <p class="likes <?= isset($_SESSION['user']) ? '' : 'openModal' ?>" id="reactions-count-<?= $d['id'] ?>">
-                <img src="/resources/img/like.png" alt="like"> <?= $d['total_reactions'] ?> reactions 
-              </p>
+              <a href="/posts/show/id:<?= $d["id"]; ?>">
+                <p class="likes <?= isset($_SESSION['user']) ? '' : 'openModal' ?>" id="reactions-count-<?= $d['id'] ?>">
+                  <img src="/resources/img/like.png" alt="like"> <?= $d['total_reactions'] ?> reactions 
+                </p>
+              </a>
```

Comment:
this do not interfere with the click made to create the reaction? i mean if i click on the lick button what will happen? open the modal to react or redirect to the /posts/show action?

Reply:
The modal does not work, it was left because I thought what work to future, but teams of design to added the modal

Reply:
not knowing that this section was going to be made, I decided to make the change I did, so, is everything ok?

Reply:
because, you make a redirection in the reactions information, you can put it outside the actions class